src/utils/constants.js

```diff
@@ -26,5 +26,5 @@ export const SPACING_SIZES = [
   'xlarge',
 ]
 
-// detect IE11 only browsers
-export const isBrowserIE11 = !!window.MSInputMethodContext && !!document.documentMode;
+// Detect IE11 only browsers in CSS using media query
+export const mediaQueryIE11 = 'all and (-ms-high-contrast: none), (-ms-high-contrast: active)';
```

Comment:
I think you can probably remove this constant and just inline the code in `flexbox.js`?

Reply:
You don't think the constant allows that media to be used elsewhere and clarifies its use? 
I can remove it if you think its best?

Reply:
I have done the change.